src/Uniphar.Dotnet.Cli/Uniphar.Dotnet.Cli.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net9.0</TargetFramework>
+		<RootNamespace>Uniphar.Dotnet.Cli</RootNamespace>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<PackAsTool>true</PackAsTool>
+		<ToolCommandName>Uniphar</ToolCommandName>
+		<PackageOutputPath>./nupkg</PackageOutputPath>
+		<PackageVersion>1.0.0</PackageVersion>
+		<Authors>Uniphar</Authors>
+		<Title>Uniphar.Dotnet.Cli</Title>
+		<Description>The repository for Uniphar.Dotnet.Cli, a .NET CLI tool designed to obtain tokens from an Entra application. Easily accessible via NuGet, this tool is built to support and simplify your development and testing processes. More features will be added in future updates.</Description>
+		<PackageProjectUrl>https://github.com/Uniphar/dotnet-cli-extension</PackageProjectUrl>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Identity.Client" Version="*" />
```

Comment:
Suggest floating on a major version, to prevent the tool breaking if there are breaking changes to the package in the future.

Reply:
Let it come... It won't break me!

Reply:
It might break the next poor soul who comes to add a new command to the tool and then gets frustrated that the thing won't compile when they haven't even changed any code yet 😅

Reply:
Fair enough!!!